### DISCUSSION TITLE
Installation impossible

### DISCUSSION THREAD

**Discussion by Mister-Wild2** [Nash115/frenchATC-for-MSFS2020#26] [Help] [answered]
Bonjour, 

Lorsque j'essaie de lancer le mod j'ai le message suivant : 

Informations météo délivrées par metar-taf.com
Impossible d'importer les modules nécessaires. Exécutez le programme 'libraries_installer.bat'.
Appuyez sur une touche pour continuer...

Alors que "libraries_installer.bat" est déja installé...

Merci pour votre aide !

**Comment by Nash115** [comment, marked as answer]
Bonjour, ce problème est déjà arrivé il y a peu de temps... Je vous renvoie vers [la discussion en question.](https://github.com/Nash115/frenchATC-for-MSFS2020/issues/19)
Cependant, si cela ne règle pas votre problème, ou que vous rencontrez des difficultés, je suis à votre disposition pour vous aider

(Je tiens tout de même à vous informer qu'une nouvelle mise à jour sera publiée car [actuellement il n'est plus possible de lancer un vol](https://github.com/Nash115/frenchATC-for-MSFS2020/issues/25))
